media/stump-volume-control/README

```diff
@@ -0,0 +1,28 @@
+very simple volume control for the window manager StumpWM
```

Comment:
It would be better to use org-mode formatting like the rest of the package READMEs, in my view.

Reply:
As detailed in my comment for the stump-radio module I do not want to go back writing org-mode or Markdown is this case. See  ttps://github.com/stumpwm/stumpwm-contrib/pull/183#discussion_r400178366

Reply:
I concur with @m-g-r.

Reply:
Thanks.